### ISSUE TITLE
ThemeAwareTranslator wrong implementation since symfony 4.2.7

### ISSUE THREAD

**Issue by Prometee** [Sylius/Sylius#10323]
**Sylius version affected**: 1.4.3

**Description**  
```
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to 
Symfony\Component\Validator\Util\LegacyTranslatorProxy::__construct() must implement interface 
Symfony\Contracts\Translation\TranslatorInterface, instance of
 Sylius\Bundle\ThemeBundle\Translation\ThemeAwareTranslator given, called in 
var/cache/prod/ContainerUaPvsqc/getValidator_BuilderService.php on line 22 and defined in 
vendor/symfony/validator/Util/LegacyTranslatorProxy.php:25
```

**Steps to reproduce**  
`composer update` and prod env

**Possible Solution**  
Use the new Interface from `symfony/contrat`

**Comment by plotek** [comment]
I think this is [`ThemeBundle`](https://github.com/Sylius/SyliusThemeBundle/issues) issue?

**Comment by pamil** [comment]
Solved in ThemeBundle v1.3.13 & v1.4.5 🎉

**Comment by Prometee** [comment]
It was very quick thanks a lot !
